concepts/05 UI Components/Chat/10 Accessibility/15 Keyboard Navigation.md

```diff
@@ -0,0 +1,14 @@
+A user can use the following keys to interact with the Chat component: 
+
+<table class="dx-table">
+    <tr>
+        <th>Key</th>
+        <th>Action</th>
+    </tr>
+    <tr>
+        <td>Enter</td>
+        <td>Enters a message.</td>
```

Comment:
The user can use Tab to move between TextArea and SendButton (SendButton is disabled when TextArea does not contain a value, in which case SendButton is not available for focus).